resources/systemd_timer.rb

```diff
@@ -27,7 +27,7 @@
 
 property :user, String, default: 'root'
 
-property :delay_after_boot, String, default: '1min'
```

Comment:
This resource is actually a backport from Chef Infra Client 16 so we need to keep the default value the same as it is in client at 1min.

Reply:
We wanted to make the OnBootsec parameter as attribute, so that it can be controlled. We are seeing a problem with the chef client timer its getting elapased ( when runinng the chef client as a timer). We do not see that problem all the time/servers but we could reproduce the elapsed issue after reducing/increasing the onbootsec parameter. So if it can be controlled by node attribute it will help us fixing this issue. We can keep the default as ( 1 min/60sec) , May be I didnt get the point here about the chef16 dependecny.

Reply:
This cookbook has a systemd resource copied from Chef Infra 16, so that users of the cookbook can use older versions of Chef Infra. In order to ease transitioning to use the resource in Chef Infra 16, this needs to be defined the same as the one there: https://github.com/chef/chef/blob/master/lib/chef/resource/chef_client_systemd_timer.rb#L65-L67

The other changes in this PR are fine, but changing the default from `'1min'` to `'60sec'` needs to be reverted.

Reply:
Sure, Reverted  let me know if this is fine. We just need that attribute to be controlled.

Reply:
Please let me know if this can be merged or need any additional modification ?

Reply:
@vineethp088 Please check https://github.com/chef-cookbooks/chef-client/pull/707#issuecomment-694400666